Add new features created by NDLA/TietoEVRY

boyum opened the pull request:
This PR adds a few more shapes to H5P Shape:

– Arrows
– 3D figures

See https://github.com/h5p/h5p-course-presentation/pull/174 for more info.

boyum committed:
Merge pull request #1 from h5p/master

EivinS committed:
Adding icons

EivinS committed:
Adding fill color to 3d-svgs

EivinS committed:
Merge pull request #2 from NDLANO/feature/adding-icons

Adding icons

kentis committed:
Update library.json

kentis committed:
Merge pull request #3 from NDLANO/bump-patch-version

Update library.json

EivinS committed:
black border on figures by default

EivinS committed:
typo in semantics

EivinS committed:
Merge branch 'master' into feature/adding-icons

EivinS committed:
Adding sharp arrows and correct scaling

EivinS committed:
Updating minorversion

EivinS committed:
update version

EivinS committed:
Merge pull request #4 from NDLANO/feature/adding-icons

Feature/adding icons

EivinS committed:
Add color-palette to shapes (#6)

* Adding palettes to shapes

* bump

Co-authored-by: EivinS <eivsel@WPF1W2VPJ.localdomain>

kentis committed:
Merge pull request #1 from NDLANO/master

Move code from NDLANO to edH5P

boyum committed:
fix lint errors

– Missing semicolon
– Expected space after "if"
– Closing curly brace appears on the same line as the subsequent block
– Fix indentation

boyum committed:
Merge pull request #7 from NDLANO/fix/lint

boyum committed:
Merge pull request #2 from NDLANO/master

kentis committed:
prefix with NDLA

kentis committed:
use ndla widgets

kentis committed:
Merge pull request #3 from NDLA-H5P/rename/prefix_ndla

prefix with NDLA

otacke commented on js/h5p-shape.js:
```diff
@@ -5,7 +5,7 @@ var H5P = H5P || {};
  */
-H5P.Shape = (function ($) {
+H5P.NDLAShape = (function ($) {
   /**
```
@boyum Are you sure you want to merge this here?

boyum committed:
change quotation marks used in svg bg images

SVG attribute's quotation marks aren't escaped, and they break the CSS `background-image` if
the value is something like `url('<svg my-attribute='' />')`. Because the SVGs use single quotation
marks, the CSS `url` value needs to use double.

boyum committed:
Merge pull request #4 from NDLA-H5P/fix/svg-quotation-marks

boyum committed:
change quotation marks used in svg bg images

boyum committed:
arrows: null check svg params

boyum committed:
Merge pull request #5 from NDLA-H5P/fix/null-check-svg

boyum committed:
bump patch version

janlindso committed:
Fixed language file and translation.

janlindso committed:
Bump

henriettemoe committed:
Use shape svg element instead of background image (#6)

for triangle, pentagon and hexagon

otacke committed:
Use regular H5PEditor.ShowWhen

otacke committed:
Bump patch version

janlindso committed:
Merge pull request #7 from NDLA-H5P/use-regular-show-when

Use regular show when
